portable/ThirdParty/GCC/Posix/port.c

```diff
@@ -420,36 +420,19 @@ static void vPortSystemTickHandler( int sig )
 
     ( void ) sig;
 
-/* uint64_t xExpectedTicks; */
-
     uxCriticalNesting++; /* Signals are blocked in this signal handler. */
 
-    #if ( configUSE_PREEMPTION == 1 )
```

Comment:
Why is `configUSE_PREEMPTION` removed here?

Reply:
It is not needed here as `xTaskIncrementTick` takes care of that.